framework/src/org/checkerframework/common/value/util/Range.java

```diff
@@ -20,6 +20,9 @@
     /** The upper bound of the interval, inclusive. */
     public final long to;
 
+    /** Should ranges take overflow into account or ignore it? */
```

Comment:
This public, static mutable state is not nice and in general we try to avoid having this. It prevents multiple checkers from running independently.
One option would be to pass in the Checker instance in the constructor and set an instance field.